[2023-12-16T14:22:58.661+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: NYC_Taxi_data_pipeline.visualization manual__2023-12-16T14:22:27.349459+00:00 [queued]>
[2023-12-16T14:22:58.669+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: NYC_Taxi_data_pipeline.visualization manual__2023-12-16T14:22:27.349459+00:00 [queued]>
[2023-12-16T14:22:58.670+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-16T14:22:58.773+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): visualization> on 2023-12-16 14:22:27.349459+00:00
[2023-12-16T14:22:58.779+0000] {standard_task_runner.py:57} INFO - Started process 368 to run task
[2023-12-16T14:22:58.781+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'NYC_Taxi_data_pipeline', 'visualization', 'manual__2023-12-16T14:22:27.349459+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/taxi_etl.py', '--cfg-path', '/tmp/tmp50n6a437']
[2023-12-16T14:22:58.783+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask visualization
[2023-12-16T14:22:58.830+0000] {task_command.py:416} INFO - Running <TaskInstance: NYC_Taxi_data_pipeline.visualization manual__2023-12-16T14:22:27.349459+00:00 [running]> on host 6abb98cbcbb2
[2023-12-16T14:22:58.892+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='islam' AIRFLOW_CTX_DAG_ID='NYC_Taxi_data_pipeline' AIRFLOW_CTX_TASK_ID='visualization' AIRFLOW_CTX_EXECUTION_DATE='2023-12-16T14:22:27.349459+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-16T14:22:27.349459+00:00'
[2023-12-16T14:23:02.632+0000] {logging_mixin.py:154} INFO - Dash is running on http://0.0.0.0:8050/
[2023-12-16T14:23:02.631+0000] {dash.py:2053} INFO - Dash is running on http://0.0.0.0:8050/

[2023-12-16T14:23:02.648+0000] {logging_mixin.py:154} INFO -  * Serving Flask app '***'
[2023-12-16T14:23:02.649+0000] {logging_mixin.py:154} INFO -  * Debug mode: on
[2023-12-16T14:47:18.242+0000] {local_task_job_runner.py:294} WARNING - State of this instance has been externally set to restarting. Terminating instance.
[2023-12-16T14:47:18.249+0000] {process_utils.py:131} INFO - Sending 15 to group 368. PIDs of all processes in the group: [378, 391, 392, 400, 402, 405, 409, 413, 417, 421, 425, 429, 433, 437, 441, 443, 447, 451, 456, 460, 463, 468, 469, 473, 477, 481, 486, 490, 492, 496, 499, 503, 507, 510, 515, 520, 368]
[2023-12-16T14:47:18.250+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 368
[2023-12-16T14:47:18.280+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=456, status='terminated', started='14:23:05') (456) terminated with exit code None
[2023-12-16T14:47:18.281+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=421, status='terminated', started='14:23:05') (421) terminated with exit code None
[2023-12-16T14:47:18.282+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=433, status='terminated', started='14:23:05') (433) terminated with exit code None
[2023-12-16T14:47:18.283+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=409, status='terminated', started='14:23:05') (409) terminated with exit code None
[2023-12-16T14:47:18.283+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=499, status='terminated', started='14:23:05') (499) terminated with exit code None
[2023-12-16T14:47:18.284+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=391, status='terminated', started='14:23:05') (391) terminated with exit code None
[2023-12-16T14:47:18.284+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=402, status='terminated', started='14:23:05') (402) terminated with exit code None
[2023-12-16T14:47:18.285+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=413, status='terminated', started='14:23:05') (413) terminated with exit code None
[2023-12-16T14:47:18.285+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=392, status='terminated', started='14:23:05') (392) terminated with exit code None
[2023-12-16T14:47:18.286+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=425, status='terminated', started='14:23:05') (425) terminated with exit code None
[2023-12-16T14:47:18.287+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=490, status='terminated', started='14:23:05') (490) terminated with exit code None
[2023-12-16T14:47:18.288+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=437, status='terminated', started='14:23:05') (437) terminated with exit code None
[2023-12-16T14:47:18.289+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=468, status='terminated', started='14:23:05') (468) terminated with exit code None
[2023-12-16T14:47:18.289+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=447, status='terminated', started='14:23:05') (447) terminated with exit code None
[2023-12-16T14:47:18.290+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=492, status='terminated', started='14:23:05') (492) terminated with exit code None
[2023-12-16T14:47:18.291+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=460, status='terminated', started='14:23:05') (460) terminated with exit code None
[2023-12-16T14:47:18.291+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=429, status='terminated', started='14:23:05') (429) terminated with exit code None
[2023-12-16T14:47:18.292+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=451, status='terminated', started='14:23:05') (451) terminated with exit code None
[2023-12-16T14:47:18.292+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=441, status='terminated', started='14:23:05') (441) terminated with exit code None
[2023-12-16T14:47:18.293+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=463, status='terminated', started='14:23:05') (463) terminated with exit code None
[2023-12-16T14:47:18.293+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=496, status='terminated', started='14:23:05') (496) terminated with exit code None
[2023-12-16T14:47:18.294+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=486, status='terminated', started='14:23:05') (486) terminated with exit code None
[2023-12-16T14:47:18.295+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=477, status='terminated', started='14:23:05') (477) terminated with exit code None
[2023-12-16T14:47:18.295+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=443, status='terminated', started='14:23:05') (443) terminated with exit code None
[2023-12-16T14:47:18.296+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=400, status='terminated', started='14:23:05') (400) terminated with exit code None
[2023-12-16T14:47:18.348+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=378, status='terminated', started='14:23:02') (378) terminated with exit code None
[2023-12-16T14:47:18.349+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=510, status='terminated', started='14:23:05') (510) terminated with exit code None
[2023-12-16T14:47:18.350+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=469, status='terminated', started='14:23:05') (469) terminated with exit code None
[2023-12-16T14:47:18.351+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=481, status='terminated', started='14:23:05') (481) terminated with exit code None
[2023-12-16T14:47:18.351+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=503, status='terminated', started='14:23:05') (503) terminated with exit code None
[2023-12-16T14:47:18.352+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=515, status='terminated', started='14:23:05') (515) terminated with exit code None
[2023-12-16T14:47:18.352+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=405, status='terminated', started='14:23:05') (405) terminated with exit code None
[2023-12-16T14:47:18.353+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=417, status='terminated', started='14:23:05') (417) terminated with exit code None
[2023-12-16T14:47:18.353+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=473, status='terminated', started='14:23:05') (473) terminated with exit code None
[2023-12-16T14:47:18.354+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=368, status='terminated', exitcode=0, started='14:22:58') (368) terminated with exit code 0
[2023-12-16T14:47:18.354+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=507, status='terminated', started='14:23:05') (507) terminated with exit code None
[2023-12-16T14:48:18.277+0000] {process_utils.py:149} WARNING - process psutil.Process(pid=520, name='airflow scheduler -- DagFileProcessorManager', status='sleeping', started='14:23:05') did not respond to SIGTERM. Trying SIGKILL
[2023-12-16T14:48:18.279+0000] {process_utils.py:86} INFO - Sending the signal 9 to group 368
[2023-12-16T14:48:18.280+0000] {process_utils.py:100} INFO - Sending the signal 9 to process 368 as process group is missing.
[2023-12-16T14:49:18.283+0000] {process_utils.py:160} ERROR - Process psutil.Process(pid=520, name='airflow scheduler -- DagFileProcessorManager', status='sleeping', started='14:23:05') (520) could not be killed. Giving up.
